tests/unit/test_cassette_sanitizer.py

```
@@ -0,0 +1,223 @@
+    do NOT need to run ``git init`` here — we just need ``tests/cassettes/`` to
+    exist and the script to be executable.
+    """
+    (tmp_path / "tests" / "cassettes").mkdir(parents=True)
+    dest = tmp_path / "tests" / "check_cassettes_clean.sh"
+    shutil.copy2(SCRIPT_PATH, dest)
+    dest.chmod(dest.stat().st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
+    return tmp_path
+
+
+def _run_guard(repo: Path) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)